### DISCUSSION TITLE
数学公式的编号显示位置距离公式太近的问题

### DISCUSSION THREAD

**Discussion by sun510001** [tangly1024/NotionNext#2007] [Q&A]
**描述bug**
使用 Notion 写 latex 公式, 在结尾处使用 \tag{} 对公式进行编号, 公式编号在 Notion 中显示正常, 但是博客中距离公式非常近, 影响了阅读理解.

**复现步骤**
在 Notion 中使用 Block equation, 编辑公式并在结尾处 添加 \tag{1}, 加入编号. 之后再博客里查看效果.

**期望的正常结果**
希望博客的公式效果应该与Notion中表现的公式效果一致, 或者公式编号尽量远离公式.

**截图**
Notion页面
![image](https://github.com/tangly1024/NotionNext/assets/32349603/fca0d750-93fa-4c98-8d97-299131e03f2d)
![image](https://github.com/tangly1024/NotionNext/assets/32349603/67166b91-764c-4c62-a855-76b49b11382a)

博客页面
![image](https://github.com/tangly1024/NotionNext/assets/32349603/2159bb3d-7c35-4db7-81c8-8c7debf0bd34)
![image](https://github.com/tangly1024/NotionNext/assets/32349603/170dcee5-1ba0-45e2-bdaa-e87b1a188a8c)

**环境**

- NotionNext版本 4.1.3
- 主题 hexo
- 部署方案 vercel
- 操作系统: windows
- 浏览器 chrome

**Comment by zfhxi** [comment]
看了https://github.com/KaTeX/KaTeX/issues/1983  ，自己调了调，没找出特别好的解决方案。目前就在global.css里加了如下代码，防止编号和公式靠得特别近：
```css
.katex-display > .katex > .katex-html > .tag {
    padding-left: 1em;
}
```

<img width="417" alt="image" src="https://github.com/user-attachments/assets/e3f29859-2aec-4065-b02f-bf1b6a462f7d" />

修改后：
<img width="536" alt="image" src="https://github.com/user-attachments/assets/82f42b30-0dff-4ece-bcf5-0439d551bb5e" />
